packages/manager/modules/log-to-customer/src/constants.js

```diff
@@ -0,0 +1,54 @@
+export const SERVICES = ['kubernetes'];
```

Comment:
Useless in that case as parameter service === constant value.
You can remove it

Reply:
For now there is only one service, but I'd prefer to keep if that way (also with the function in the resolve), so the architecture and the way to retrieve service is already in place and known